app/helpers/application_helper/button/test_button.rb

```diff
@@ -0,0 +1,14 @@
+class ApplicationHelper::Button::TestButton < ApplicationHelper::Button::Basic
+  needs_record
+  def initialize(view_context, view_binding, instance_data, props)
+    super(view_context, view_binding, instance_data, props)
+    @method = props[:options][:method]
+  end
+
+  def skip?
+    return true unless @method
+    return !@record.send("supports_#{@method}?")
```

Comment:
Maybe test `record.respond_to?("supports_#{@method}")` instead of exception handling which is more generic? But not sure it's worth changing. @himdel ? @mzazrivec ?